packages/terra-action-footer/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ ChangeLog
 Unreleased
 ----------
 
+1.27.0 - (October 23, 2018)
+------------------
+### Changed
+* Styling improvements
```

Comment:
Go ahead and put this in the Unlreleased section above; we will move it and create a new section when the release occurs.

Reply:
Done in my latest push.